exercises/sum.cpp

```diff
@@ -1,8 +1,12 @@
-/*
-  Write a program that takes as input two numbers and print the sum.
+#include <iostream>
 
-  Output:
-  Insert the first number: 1
-  Insert the second number: 2
-  Sum: 3
-*/
+using namespace std;
+
+int main(){
+	int a, b;
+	cout<<"Inserire primo addendo\n";
+	cin>>a;
+	cout<<"Inserire secondo addendo\n";
+	cin>>b;
+	cout<<"La somma e': "<<(a+b)<<"\n";
+}
```

Comment:
Manca il return 0;
Anche se il programma funziona comunque ritornare il valore 0 a fine esecuzione ti serve per assicurarti che se l'esecuzione termina proprio con valore 0 alloral'esecuzione ha terminato correttamente. 
Caso contrario (presenza di errori a runtime) si avrà un exit code diverso.
Meglio usare una macro per i valori handcoded , come lo 0 nel return.
 Questo perchè le macro create con #define non una variabile perchè occupa memora, il simbolo di macro viene sostituito con il valore reale in fase di compilazione, per info: https://www.arduino.cc/reference/en/language/structure/further-syntax/define/:
```diff
+ #define EXIT_SUCCESS 0
....
+ return EXIT_SUCCESS;
- return 0;
```